registry/Scripts/script.sh

```diff
@@ -126,6 +127,12 @@ fetchCredentials() {
             "${secret}?api-version=2016-10-01" -H "Authorization: Bearer ${TOKEN}" | jq -r .value)
         htpasswd -Bb .htpasswd ${SECRET_NAME} ${SECRET_VALUE}
     done
+
+    if [ ! -s .htpasswd ]; then
+        echo ".htpasswd file is empty."
+        return 1
```

Comment:
I would do `return $ERR_MISSING_USER_CREDENTIAL` here instead of the caller, just in case we want to add more error conditions in the future (ex curl failures)

Reply:
Updated.